src/actions/user.ts

```diff
@@ -98,6 +100,50 @@ export const signUpUser = (
   }
 }
 
+export const deleteAccount = () => {
+  return async (dispatch: AppDispatch) => {
+    return authRequestWithDispatch({
+      dispatch,
+      method: 'DELETE',
+      endpoint: 'delete_users',
+      types: [SET_USER_REQUEST, SET_USER_SUCCESS, SET_USER_FAILURE],
+    })
+  }
+}
+
+export const sendResetPassEmail = (email: string) => {
+  return async (dispatch: AppDispatch) => {
+    dispatch({ type: SET_USER_REQUEST })
+    try {
+      await sendPasswordResetEmail(auth, email)
+      dispatch({ type: SET_USER_SUCCESS })
+    } catch (error: any) {
+      dispatch({ type: SET_USER_FAILURE, payload: error })
+    }
+  }
+}
+
+export const updateUser = (userInfo: User) => {
+  return async (dispatch: AppDispatch) => {
+    await authRequestWithDispatch({
+      dispatch,
+      endpoint: 'update_user',
+      method: 'PATCH',
+      types: [SET_USER_REQUEST, SET_USER_SUCCESS, SET_USER_FAILURE],
+      data: {
+        name: userInfo.name,
+        major: userInfo.major,
+        email: userInfo.email,
+        pid: userInfo.pid,
+        onboarded: userInfo.onboarded,
+        photoURL: userInfo.photoURL,
+        borderURLColor: userInfo.borderURLColor,
+      },
+    })
+    // dispatch({ type: SET_USER_SUCCESS })
+  }
+}
```

Comment:
Can you change the `await` to `return` to make it a little more concise?  Also can you get rid on the comment.